ledger-core/virtual/execute/execute_test.go

```diff
@@ -563,4 +564,99 @@ func TestSendVStateReportWithMissingState_IfConstructorWasInterruptedBeforeRunne
 		close(vStateReportRecv)
 		t.Error("timeout")
 	}
+
+	mc.Finish()
+}
+
+func TestSMExecute_StopWithoutMessagesIfPulseChangedBeforeOutgoing(t *testing.T) {
+	t.Log("C5101")
+	defer testutils.LeakTester(t,
+		goleak.IgnoreTopFunction("github.com/insolar/assured-ledger/ledger-core/runner.(*worker).Run.func1"),
+		goleak.IgnoreTopFunction("github.com/insolar/assured-ledger/ledger-core/conveyor/smachine.startChannelWorkerUnlimParallel.func1"),
+	)
+
+	var (
+		mc  = minimock.NewController(t)
+		ctx = context.Background()
+
+		class     = gen.UniqueGlobalRef()
+		caller    = gen.UniqueGlobalRef()
+		objectRef = gen.UniqueGlobalRef()
+
+		catalog     object.Catalog         = object.NewLocalCatalog()
+		authService authentication.Service = authentication.NewServiceMock(t)
+	)
+
+	slotMachine := slotdebugger.New(ctx, t)
+	slotMachine.PrepareMockedMessageSender(mc)
+	slotMachine.PrepareMockedRunner(ctx, mc)
+	slotMachine.AddInterfaceDependency(&catalog)
+	slotMachine.AddInterfaceDependency(&authService)
+
+	var vStateReportRecv = make(chan struct{})
+	checkMessage := func(msg payload.Marshaler) {
+		res, ok := msg.(*payload.VStateReport)
+		require.True(t, ok)
+		assert.Equal(t, objectRef, res.Object)
+		assert.Equal(t, payload.Ready, res.Status)
+		assert.Equal(t, int32(0), res.OrderedPendingCount)
+		assert.Equal(t, int32(0), res.UnorderedPendingCount)
+		assert.Equal(t, []byte("213"), res.ProvidedContent.LatestDirtyState.State)
```

Comment:
I would assign '[]byte("213")' to variable and reuse it

Reply:
done